docs/upgrade.html

```diff
@@ -37,6 +37,8 @@ <h5><a id="upgrade_300_notable" href="#upgrade_300_notable">Notable changes in 3
     <li>The deprecated <code>kafka-preferred-replica-election</code> command line tool was removed. Please use <code>kafka-leader-election</code> instead.</li>
     <li>The deprecated <code>ConfigEntry</code> constructor was removed (<a href="https://issues.apache.org/jira/browse/KAFKA-12577">KAFKA-12577</a>).
         Please use the remaining public constructor instead.</li>
+    <li>Kafka Streams no longer has a compile time dependency on "connect:json" module (<a href="https://issues.apache.org/jira/browse/KAFKA-5146">KAFKA-5146</a>).
```

Comment:
Not: For the top level docs, it might be ok to remove the link to the ticket?

Reply:
In docs/upgrade.html they all reference a Kafka jira ticket with the hyperlink. In docs/streams/upgrade-guide.html they all reference a KIP, but there were previous upgrades referencing a jira ticket with the hyperlink. Seems consistent to keep hyperlink in both files.